_scripts/base-docker.sh

```diff
@@ -7,7 +7,30 @@ for d in ./packages/*/ ; do
   (cd "$d" && mkdir -p config && cp ../version.json . && cp ../version.json config)
 done
 
-npm i lerna
-npx lerna bootstrap --hoist pm2
-npx lerna run --stream build
-npx lerna exec --stream --concurrency 2 --no-bail -- npm prune --production
+# `npx yarn` because `npm i -g yarn` needs sudo
+npx yarn install
+SKIP_PREFLIGHT_CHECK=true npx yarn workspaces foreach --topological run build
+rm -rf node_modules
+rm -rf packages/*/node_modules
+npx yarn workspaces focus --production \
```

Comment:
Does this kind of serve a similar purpose as `npm ci` for minimal install?

Reply:
yes, `focus --production` is the way to do the equivalent of `npm ci --production`. Plain `yarn install` doesn't have that option.